chpl/chpl-service/src/main/java/gov/healthit/chpl/entity/MeaningfulUseAccurateAsOfEntity.java

```diff
@@ -0,0 +1,88 @@
+package gov.healthit.chpl.entity;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Entity
+@Table(name="muu_accurate_as_of_date")
+public class MeaningfulUseAccurateAsOfEntity implements Cloneable, Serializable {
+	private static final long serialVersionUID = -1463562876433962214L;
+	
+	@Id 
+	@GeneratedValue(strategy = GenerationType.IDENTITY)
+	@Basic(optional = false)
+	@Column(name = "muu_accurate_as_of_date_id", nullable = false)
+	private Long id;
+	
+	@Basic(optional=false)
+	@Column(name = "Accurate_as_of_date", updatable = true, nullable = false)
+	private Date accurateAsOfDate;
```

Comment:
Case must not matter here although I would have thought on Windows it would. Make "accurate" in the column name lowercase for consistency.